client/src/nv_ingest_client/primitives/tasks/extract.py

```diff
@@ -32,26 +32,41 @@
 ADOBE_CLIENT_SECRET = os.environ.get("ADOBE_CLIENT_SECRET", None)
 
 _DEFAULT_EXTRACTOR_MAP = {
+    # Image files
     "bmp": "image",
-    "csv": "pandas",
-    "docx": "python_docx",
-    "excel": "openpyxl",
-    "html": "txt",
     "jpeg": "image",
     "jpg": "image",
-    "parquet": "pandas",
-    "pdf": "pdfium",
     "png": "image",
-    "pptx": "python_pptx",
-    "text": "txt",
     "tiff": "image",
-    "txt": "txt",
-    "xml": "lxml",
-    "mp3": "audio",
-    "wav": "audio",
+    # Text files
+    "html": "txt",
     "json": "txt",
     "md": "txt",
     "sh": "txt",
+    "text": "txt",
+    "txt": "txt",
+    "log": "txt",
+    "yaml": "txt",
+    "yml": "txt",
+    "ini": "txt",
+    "conf": "txt",
+    "cfg": "txt",
+    "tex": "txt",
+    "rst": "txt",
+    "adoc": "txt",
+    # Data files
+    "csv": "pandas",
+    "parquet": "pandas",
+    "excel": "openpyxl",
+    # Document files
+    "docx": "python_docx",
+    "pdf": "pdfium",
+    "pptx": "python_pptx",
+    # Audio files
+    "mp3": "audio",
```

Comment:
Have you tested with mp3s? I can't remember if we have an extract path for this in the audio extractor stage.